docs/integrations/sources/stripe-migrations.md

```diff
@@ -0,0 +1,6 @@
+# Stripe Migration Guide
+
+## Upgrading to 4.0.0
+
+A major update of most streams to support event-based incremental sync mode.
+For a smooth migration, a schema refresh is needed as cursor fields are changed for many streams.
```

Comment:
> For a smooth migration
If the refresh is required, please say so unambiguously.  What will happen if they don't?

Reply:
updated